lib/reference/properties.js

```diff
@@ -370,6 +370,16 @@ const longhandSubPropertiesOfShorthandProperties = new Map([
 			'transition-delay',
 		]),
 	],
+	[
+		'inset',
+		new Set([
+			// prettier-ignore
+			'top',
+			'right',
+			'bottom',
+			'left',
+		]),
+	],
```

Comment:
[suggest] Could you move `inset` right before `list-style` for alphabetical order?